components/tests/integration_test.cpp

```diff
@@ -109,22 +109,32 @@ int authenticate_json(const std::string& json_file, const bool compute_validatio
 
             if (n_vertices != expected_vertices) {
                 spdlog::error(
-                    "Violating Authenticate for vertices {}!={}",
+                    "Violating Authenticate in mesh `{}` for vertices {} != {}",
+                    meshes[i],
                     n_vertices,
                     expected_vertices);
                 return 2;
             }
             if (n_edges != expected_edges) {
-                spdlog::error("Violating Authenticate for edges {}!={}", n_edges, expected_edges);
+                spdlog::error(
+                    "Violating Authenticate in mesh `{}` for edges {} != {}",
+                    meshes[i],
+                    n_edges,
+                    expected_edges);
                 return 2;
             }
             if (n_faces != expected_faces) {
-                spdlog::error("Violating Authenticate for faces {}!={}", n_faces, expected_faces);
+                spdlog::error(
+                    "Violating Authenticate in mesh `{}` for faces {} != {}",
+                    meshes[i],
+                    n_faces,
+                    expected_faces);
                 return 2;
             }
             if (n_tetrahedra != expected_tetrahedra) {
                 spdlog::error(
-                    "Violating Authenticate for tetrahedra {}!={}",
+                    "Violating Authenticate in mesh `{}` for tetrahedra {} != {}",
+                    meshes[i],
                     n_tetrahedra,
                     expected_tetrahedra);
                 return 2;
```

Comment:
while we're at it could we remove these random numbers with an enum or something?
also aren't we trying to use wmtk::logger?

Reply:
I can switch to wmtk::logger, yes. Actually, it would be better if we could have catch2 checks with custom messages but AFAIK they don't exist yet.

Please feel free to take a look also at the rest of the code. As I am already at it, I can clean it up a bit more.